context/app/static/js/components/Detail/Citation/Citation.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { LightBlueLink } from 'js/shared-styles/Links';
+import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+import SectionContainer from 'js/shared-styles/sections/SectionContainer';
+import { FlexPaper } from './style';
+
```

Comment:
Include a comment which includes the citation format we're using?

Reply:
The function name now has "NLM".